sdk/communication/azure-communication-chat/src/test/java/com/azure/android/communication/chat/ChatClientTestBase.java

```diff
@@ -174,6 +178,15 @@ public boolean checkParticipantsListContainsParticipantId(List<ChatParticipant>
         return false;
     }
 
+    static void awaitOnLatch(CountDownLatch latch, String method) {
+        long timeoutInSec = 2;
+        try {
+            latch.await(timeoutInSec, TimeUnit.MINUTES);
+        } catch (InterruptedException e) {
+            assertFalse(true, method + " didn't complete within " + timeoutInSec + " minutes");
```

Comment:
Is the original purpose to use 2 seconds or 2 minutes delay here?
"timeoutInSec" conflicts with "minutes" in error message.

Reply:
my bad, I meant to name the variable `timeoutInMin`

Reply:
Let me do the test name cleanup in a follow up.